assets/css/style.css

```diff
@@ -95,4 +95,5 @@ body {
     width: 150px;
     height: 150px;
   }
+  .a:active { background-color: green; color: white; padding: 10px 25px; text-align: center; text-decoration: none; display: inline-block}
```

Comment:
Hey @Shivadubal26 !
What does this change do?
Please explain.

Reply:
it will act on all active  href link which user will click

Reply:
Please send screenshot with changes from your localhost. So, that I can see how it looks!